geometry_msgs/Point p_left
geometry_msgs/Point p_right
geometry_msgs/Point object 
